sympy/functions/elementary/tests/test_trigonometric.py

```diff
@@ -79,6 +79,7 @@ def test_sin():
     assert sin(pi*e/2) == 0
     assert sin(pi*k) == 0
     assert sin(pi*k).subs(k, 3) == sin(pi*k/2).subs(k, 6)  # issue 8298
+    assert sin(2*k*pi + 4) == sin(4) # issue 17976
```

Comment:
Write `pi*2*k` although that is not different than you wrote but to match the pattern of test cases.

Reply:
ok, i will correct this.